lib/android/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.airbnb.android.react.maps" >
+    <uses-sdk android:minSdkVersion="16" />
```

Comment:
It's necessary for intelliJ / Android Studio to give you the proper warnings...

...it's not related to this PR, however :P